### ISSUE TITLE
Attachment vertices don't draw correct shape 

### ISSUE THREAD

**Issue by mspanish** [pixijs/pixi-spine#160]
I'm trying to use the vertices from either a bounding box or mesh (I've tried both and get the same results) so that I can create hitAreas on the shapes. For testing I'm drawing Pixi graphics just to see where the hitArea is drawing - and for some reason I'm getting the graphics in sort of an inverted shape. I'm attaching the graphics to the container where the spine animation is loaded, and don't have flipY or anything like that set.  Here you can see what I have in Spine, then what draws when I try using the vertices.

My code for getting them:

`vertices = sub.skeleton.findSlot("port_triangle").attachment.vertices;`

Then I use them to draw:

```
var graphics = new PIXI.Graphics(); 
graphics.beginFill(0x0000ff); 
graphics.drawPolygon([
    v[0],v[1],
    v[2],v[3],
    v[4],v[5],
]);
graphics.endFill();
subContainer.addChild(graphics);
```
Seems I can't upload an image, I'll try again later

**Comment by mspanish** [comment]
![pix2](https://cloud.githubusercontent.com/assets/206150/23435606/58cc41ae-fdb4-11e6-862d-cb195b7bc4b2.png)

**Comment by mspanish** [comment]
What I had to do in order for these shapes to draw exactly over the meshes, was to put a negative value on all of the y coordinates, and pass the bone.worldX and bone.worldY and set the graphic value to those values.

```
var graphics = new PIXI.Graphics(); 
graphics.beginFill(0x0000ff); 
graphics.drawPolygon([
    v[2],-v[3],
    v[4],-v[5],
    v[0],-v[1],
]);
graphics.endFill();
graphics.x = x;
graphics.y = y;
subContainer.addChild(graphics);
```

![pix3](https://cloud.githubusercontent.com/assets/206150/23436812/55c21912-fdb8-11e6-84c6-beee19fd28a2.png)

**Comment by ivanpopelyshev** [comment]
Yes, actually, PIXI computes and draws everything Y-reversed, there's even a constant somewhere, "Bone.yDown". So vertices in "attachment" and vertices in the slot are differ by a sign :)

**Comment by mspanish** [comment]
Ah great to know - one question I have - will using shapes w/ no fill be bad for memory? For my hitAreas that are on sprites, I just add the geometric Pixi shape without actually drawing a graphic - can I use that on a non sprite, and still position it? Something like:

```
function drawGeoPolygon(v,x,y) {
     var poly = new PIXI.Polygon(
          v[0],-v[1],
          v[2],-v[3],
          v[4],-v[5],
          v[6],-v[7],
            );
  poly.x = x;
  poly.y = y;   
};
```

**Comment by ivanpopelyshev** [comment]
No, geometry shapes cant be part of pixi stage, and they dont have transform, (no x,y). No, it wont be bad for memory, just dont forget to destroy() spine instances you don't need anymore. When you call 

```js
spineObj.destroy({ children: true});
```

It is destroying all graphics and meshes inside of it. That memory leak is very small, but after huge amount of time it'll be noticable.

**Comment by mspanish** [comment]
Thanks very much @ivanpopelyshev - I appreciate the help very much.
